scripts/processAddresses.ts

```diff
@@ -43,6 +43,27 @@ async function main() {
         addresses[keyName]=addressesFile["tokens"][baseAsset]
     }
 
+    // Add base assets that exist in the network that was NOT selected to the json
+
+    // get NOT the network name
+    let notNetwork = network == "mainnet"? "goerli": "mainnet";
+    // read in address file and parse
+    let notNetworkRawdata = fs.readFileSync("addresses/"+notNetwork+".json").toString();
+    let notNetworkAddressesFile = JSON.parse(notNetworkRawdata);
+    let notNetworkBaseAssets = [];
+    // store base assets listed in NOT network json
+    for (const trancheListKey in notNetworkAddressesFile["tranches"]) {
+        notNetworkBaseAssets.push(trancheListKey);
+    }
+
+    // add base asset tokens that are NOT already added
+    for (const notNetworkBaseAsset of notNetworkBaseAssets) {
+        const keyName = notNetworkBaseAsset+"Address"
+        if (!addresses.hasOwnProperty(keyName)){
+            addresses[keyName]="0x0";
```

Comment:
Minor: change this to `0x0000000000000000000000000000000000000000`.

`0x0` is problematic for `ethers` since it tries to do an ENS lookup on anything that doesn't match a full address regex.

Reply:
```suggestion
            addresses[keyName]="0x0000000000000000000000000000000000000000";
```